lib/_http_client.js

```diff
@@ -723,12 +710,14 @@ function requestOnPrefinish() {
 function emitFreeNT(req) {
   req.emit('close');
   if (req.res) {
-    req.res.emit('close');
+    // TODO(dnlup): is this still needed since using autoDestroy: true?
+    req.res.destroy();
```

Comment:
I need to to some further testing to see if this is needed after setting `autoDestroy: true`.